packages/services/src/builder/index.ts

```diff
@@ -90,7 +90,7 @@ class BuildManager {
         throw new ServerConnection.ResponseError(response, 'Build aborted');
       }
       if (response.status !== 200) {
-        let message = 'Build failed with ${response.status)`, please run `jupyter lab build` on the server for full output';
+        let message = 'Build failed with ${response.status}`, please run `jupyter lab build` on the server for full output';
```

Comment:
You also need to fix the string to be surrounded by backticks (and remove the stray backtick from within the string).